api/app/models/flask_config.py

```diff
@@ -5,6 +5,7 @@ class Config(object):
     DEBUG = False
     TESTING = False
     DATABASE_URI = os.environ.get("DATABASE_URL")
+    SECRET_KEY = os.environ.get("SECRET_KEY")
 
 
```

Comment:
@bnorquist, I don't think this config file belongs here in the models directory. Can you move it to: `app/config.py`. so we can use it as an import via `app.config`